arm-servicebus/2014-09-01/swagger/servicebus.json

```diff
@@ -566,153 +572,2046 @@
+          "type": "string",
+          "description": "Resource Manager type of the resource."
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the subscription."
+        },
+        "properties": {
+          "$ref": "#/definitions/SubscriptionProperties"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "description": "Parameters supplied to the CreateOrUpdate Subscription operation."
+    },
+    "SubscriptionProperties": {
+      "properties": {
+        "AccessedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Last time a there was a receive request to this subscription."
+        },
+        "AutoDeleteOnIdle": {
+          "type": "string",
+          "format": "time-span",
+          "description": "TimeSpan idle interval after which the topic is automatically deleted. The minimum duration is 5 minutes."
+        },
+        "AvailabilityStatus ": {
+          "type": "string",
+          "description": "Entity availability status for the topic.",
+          "enum": [
+            "Available",
+            "Limited",
+            "Renaming",
+            "Restoring",
+            "Unknown"
+          ],
+          "x-ms-enum": {
+            "name": "NamespaceType",
+            "modelAsString": false
+          }
+        },
+        "CreatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Exact time the message was created."
+        },
+        "DefaultMessageTimeToLive": {
+          "type": "string",
+          "format": "time-span",
+          "description": "Default message time to live value. This is the duration after which the message expires, starting from when the message is sent to Service Bus. This is the default value used when TimeToLive is not set on a message itself."
+        },
+        "EnableBatchedOperations": {
+          "type": "boolean",
+          "description": "Value that indicates whether server-side batched operations are enabled.."
+        },
+        "EnableDeadLetteringOnFilterEvaluationExceptions": {
+          "type": "boolean",
+          "description": "Value that indicates if a subscription has dead letter support on Filter evaluation exceptions."
+        },
+        "EnableDeadLetteringOnMessageExpiration": {
+          "type": "boolean",
+          "description": "Value that indicates if a subscription has dead letter support when a message expires."
+        },
+        "ExtensionData": {
+          "type": "object",
+          "description": "Structure that contains extra data."
+        },
+        "ForwardDeadLetteredMessagesTo": {
+          "type": "string",
+          "description": "Path to the recipient to which the dead lettered messages are forwarded."
+        },
+        "ForwardTo": {
+          "type": "string",
+          "description": "Path to the recipient to which the dead lettered messages are forwarded."
```

Comment:
What is the difference between "ForwardDeadLetteredMessagesTo" and "ForwardTo" as they both have the same description
> "Path to the recipient to which the dead lettered messages are forwarded.""Path to the recipient to which the dead lettered messages are forwarded."

Please make sure that the description is accurate.